cmp/grid/ColChooser.js

```diff
@@ -30,6 +30,7 @@ export class ColChooser extends Component {
             items: [
                 leftRightChooser({model: lrModel, height: 300}),
                 toolbar(
+                    leftRightChooserFilter({model: lrModel, fields: ['text'], anyMatch: true}),
```

Comment:
A bit worried that the reset button might now be interpreted as "reset the filter" vs. "reset my custom columns" - especially as we don't now do any confirm on reset. (Note, do not think that we want to add a confirm at this point - they can be overkill sometimes.)

Would try moving Reset to the other side of the filler, next to the other buttons. Does that look natural?